connector-runtime/pom.xml

```diff
@@ -41,6 +41,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Shouldn't this come as transitive dependency from the starter (and thus not be necessary?)